### PR TITLE
Bug 1254169 - Migrate Keyboard to use L20n

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1254169

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=58802e668e020ff856da13c4aa97ced6429cbdbd

File: apps/keyboard/js/settings/layout_item_view.js [hunk 1]
```diff
@@ -53,34 +53,34 @@ LayoutItemView.prototype.start = function() {
...
-      loadedSize: loadedSizeData.size,
-      loadedSizeUnit: loadedSizeData.localizedUnit,
-      totalSize: totalSizeData.size,
-      totalSizeUnit: totalSizeData.localizedUnit
-    });
+
+  Promise.all([
+    this._getHumanFileSize(this._model.downloadLoadedSize),
+    this._getHumanFileSize(this._model.downloadTotalSize)
+  ]).then(([loadedSize, totalSize]) => {
```
Comment (timdream) [inline apps/keyboard/js/settings/layout_item_view.js]: es6 will unfortunately break Keyboard demo running on the Web, which runs keyboard with code in the source tree. Could you use `values[0]` and `values[1]` here?

File: apps/keyboard/js/settings/layout_item_view.js [hunk 2]
```diff
@@ -53,34 +53,34 @@ LayoutItemView.prototype.start = function() {
...
+    console.log('updating');
+    console.log(loadedSize);
```
Comment (timdream) [inline apps/keyboard/js/settings/layout_item_view.js]: Extra `console.log` should be removed.

File: apps/keyboard/js/settings/layout_item_view.js [hunk 3]
```diff
@@ -102,16 +102,17 @@ LayoutItemView.prototype._updateUI = function() {
...
-          loadedSize: loadedSizeData.size,
-          loadedSizeUnit: loadedSizeData.localizedUnit,
-          totalSize: totalSizeData.size,
-          totalSizeUnit: totalSizeData.localizedUnit
-        });
+      this._statusEl.dataset.l10nId = 'downloadingStatus2';
+      Promise.all([
+        this._getHumanFileSize(item.downloadLoadedSize),
+        this._getHumanFileSize(item.downloadTotalSize)
+      ]).then(([loadedSizeData, totalSizeData]) => {
```
Comment (timdream) [inline apps/keyboard/js/settings/layout_item_view.js]: Same here.

File: apps/keyboard/js/settings/layout_item_view.js (hunk 1 above)
Comment (zbraniecki) [inline apps/keyboard/js/settings/layout_item_view.js]: so, are fat arrow functions also a no-go?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=799ed19d35df344271175531e56293bcd1edca4b

**Pull request merged**